fastlane/lib/fastlane/actions/crashlytics.rb

```diff
@@ -64,7 +66,6 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :crashlytics_path,
                                        env_name: "CRASHLYTICS_FRAMEWORK_PATH",
                                        description: "Path to the submit binary in the Crashlytics bundle (iOS) or `crashlytics-devtools.jar` file (Android)",
-                                       default_value: Dir["./Pods/iOS/Crashlytics/Crashlytics.framework"].last || Dir["./**/Crashlytics.framework"].last,
```

Comment:
It always set to the "Crashlytics.framework" directory if the directory existed, even if I build for android platform. so I move it to `generate_ios_command`.